app/qml/project/MMProjectWizardPage.qml

```diff
@@ -151,4 +132,32 @@ MMPage {
       }
     }
   }
+
+  MM.FieldsModel {
+    id: fieldsModel
+
+    onNotifyError: function( message ) {
+      __notificationModel.addError( message )
+    }
+  }
+
+  ListModel {
+    id: typesmodel
+
+    ListElement { text: "Text"; type: "TextEdit" }
+    ListElement { text: "Date&time"; type: "DateTime" }
+    ListElement { text: "Number"; type: "Range" }
+    ListElement { text: "Checkbox"; type: "CheckBox" }
+    ListElement { text: "Photo"; type: "ExternalResource" }
+  }
+
+  function indexFromWidgetType( widgetType ) {
+    for ( let i = 0; i < typesmodel.count; i++ ) {
+      let item = typesmodel.get(i)
+      if ( widgetType === item.type ) {
+        return i
+      }
+    }
+    return -1
+  }
```

Comment:
wondering if more generic version of this function shouldn't be part of comboboxinput ? since we will probably need to use it in other situations too?